clients/restclient/rest_client.go

```diff
@@ -18,13 +22,16 @@ type RestClient struct {
 
 // NewRestClient creates a new Rest client
 func NewRestClient(host, org, space string, rt http.RoundTripper, jar http.CookieJar, tokenFactory baseclient.TokenFactory) RestClientOperations {
-	t := baseclient.NewHTTPTransport(host, getRestURL(org, space), getRestURL(baseclient.EncodeArg(org), baseclient.EncodeArg(space)), rt, jar)
+	t := baseclient.NewHTTPTransport(host, restBaseURL, restBaseURL, rt, jar)
```

Comment:
Shouldn't this be the same as before? If you change it, the calls to the backend might not contain the org and space parameters or ?

Reply:
Actually, the RestClient is used only by purge command which has different URL withtout org and space names in it.